### ISSUE TITLE
Run development with nodemon?

### ISSUE THREAD

**Issue by prashantidealittechno** [Osedea/nodock#113]
How can we run the project with nodemon so we do not have to restart the containers for small changes or fixes in files?

**Comment by philtrep** [comment]
@prashantidealittechno Absolutely, if you can replace the entrypoint for the node service from `run-nodock "node index.js"` to whatever you wish. You'd just have to make sure that you've installed the program you want to run.

In your case, the best would probably be to use the scripts section of the `package.json` to call `nodemon index.js` and call the npm script from the entrypoint.

Let me know if that works for you.
